src/server/api/user.ts

```diff
@@ -47,18 +53,18 @@ const urlSchema = Joi.object({
   shortUrl: Joi.string()
     .custom((url: string, helpers) => {
       if (!isValidShortUrl(url)) {
-        return helpers.message({ message: 'Short url format is invalid.' })
+        return helpers.message({ custom: 'Short url format is invalid.' })
       }
       return url
     })
     .required(),
   longUrl: Joi.string().custom((url: string, helpers) => {
     if (!isHttps(url)) {
-      return helpers.message({ message: 'Long url must start with https://' })
+      return helpers.message({ custom: 'Long url must start with https://' })
     }
     if (blacklist.some((bl) => url.includes(bl))) {
       return helpers.message({
-        message: 'Creation of URLs to link shortener sites prohibited.',
+        custom: 'Creation of URLs to link shortener sites prohibited.',
```

Comment:
Did something break with the `message` key? or were these changes made for consistency etc??

Reply:
my bad, just saw the PR description and realized it's to pass the error to frontend. Resolving

Reply:
Yeah apparently if you use Joi's custom method the message has to be in key 'custom'